app/views/users/search.js.coffee

```diff
@@ -0,0 +1 @@
+$('#users').html("<%= j render @users %>")
```

Comment:
My eyes!  It burns!  I haven't personally used UJS & server sent javascript for a lot of years.  But it's a terrible thing which points to some of the most terrible smells in our codebase.  Here's a great article about it:

https://reinteractive.net/posts/195-rails-server-sent-javascript-considered-harmful

It is one of those ultra magical things... I sent a request to the server, and voila!  Magical things happen to my page.  Except when you're trying to change some of this functionality.